src/accounts/accounts.service.ts

```diff
@@ -95,4 +100,22 @@ export class AccountsService {
     const account = await this.findOne(address);
     return account.getSubsidy();
   }
+
+  public async freezeSecondaryAccounts(opts: TransactionBaseDto): ServiceReturn<void> {
+    const { signer, webhookUrl } = opts || {};
+    const { polymeshService, transactionsService } = this;
+
+    const { freezeSecondaryAccounts } = polymeshService.polymeshApi.accountManagement;
```

Comment:
```suggestion
    const { 
        polymeshService: { 
            polymeshApi: { 
                accountManagement: { 
                  freezeSecondaryAccounts
                }
             }
          }, transactionsService } = this;
```

Reply:
You don't need to destructure all of the way

```
 polymeshService: { polymeshApi }
```

or

```
 polymeshService: { polymeshApi: { accountManagement } }
```

would be OK

I think its better avoid to avoid the double `const` declaration here.